### PR TITLE
fix : added hyperlinks

### PR DESCRIPTION
**Pull request by Sharmaryan**
Screenshot (34)

added hyperlink

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (yashpatel1797) [inline index.html:6]: add author meta tag as well.

**Review by yashpatel1797 — APPROVED** [review]
looks good to merge.

File: index.html (hunk 1 above)
Comment (Sharmaryan) [reply index.html:6]: done

**Comment by sharath-io** [comment]
Looks good for merging.

**Review by sharath-io — APPROVED** [review]
Looks good for merging.

**Pull request merged**